lib/toolshed/unix.ex

```diff
@@ -26,7 +26,7 @@ defmodule Toolshed.Unix do
   end
 
   @doc """
-  Run a regular expression on a file and print the matching lines.
+  Run a regular expression on a file and print the matching lines.Incase there is no match an empty string is returned.
```

Comment:
I don't think this comment is correct. Nothing should be printed if there are no matching lines. If something is happening, then I'm curious to investigate more. If not, I think the easiest fix is to keep the original text here.